WireMock/wwwroot/scripts/helper/createElement.ts

```
@@ -0,0 +1,66 @@
+            for (const key in options.attributes) {
+                if (options.attributes.hasOwnProperty(key)) {
+                    element.setAttribute(key, options.attributes[key]);
+                }
+            }
+        }
+    }
+
+    function setStyleIfApply() {
+        if (options.style) {
+            for (const key in options.style) {
+                if (options.style.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)